src/v/datalake/tests/arrow_writer_test.cc

```diff
@@ -112,14 +119,177 @@ test_timestamp: timestamp[us] not null
 test_timestamptz: timestamp[us, tz=UTC] not null
 test_string: string not null
 test_uuid: fixed_size_binary[16] not null
-test_fixed: fixed_size_binary[100] not null
+test_fixed: fixed_size_binary[11] not null
 test_binary: binary not null
 test_struct: struct<nested_bool: bool not null, nested_int: int32 not null, nested_long: int64 not null> not null
 test_list: list<element: string not null> not null
 test_map: map<string, int64> not null)";
     EXPECT_EQ(schema->ToString(), expected_schema);
 }
 
+TEST(ArrowWriterTest, TranslatesData) {
+    datalake::arrow_schema_translator schema_translator(
+      test_schema(iceberg::field_required::no));
+
+    for (int i = 0; i < 5; i++) {
+        auto data = iceberg::tests::make_value(
+          iceberg::tests::value_spec{
+            .forced_fixed_val = iobuf::from("Hello world")},
+          test_schema(iceberg::field_required::no));
+        schema_translator.add_data(std::move(data));
+    }
+
+    std::string result_string = schema_translator.take_chunk()->ToString();
+
+    std::string expected
+      = "-- is_valid: all not null\n-- child 0 type: bool\n  [\n    false,\n   "
```

Comment:
can we put this into a raw c++ string literal to make this more readable?

Reply:
There's a lot of newlines in there. Do you think it would make sense to pull it into a separate file so it doesn't make this one hard to read?

Reply:
Yea we sometimes do that, e.g. https://github.com/redpanda-data/redpanda/blob/2d6a7ba4abeb0c0a79dd4317088047a20b6f5e0d/src/v/iceberg/tests/test_schemas.cc#L66-L172

Maybe we could just add it there? Or a separate file just for parquet (no strong preference)